Automattic/wp-calypso
# Integrate lightbox component to feature and simple card

#### Proposed Changes

* Create `useItemLightbox` custom hook for managing lightbox state.
* Integrate lightbox component to featured card and simple card using `useItemLightbox`.
* Remove temporary hint button for displaying lightboxes.

#### Testing Instructions
* Run `git fetch && git checkout add/lightbox-to-feature-and-product-card`
  * Run `yarn start-jetpack-cloud`
* Go to http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1 or use the jetpack cloud live link and goto `/pricing?flags=jetpack/pricing-page-rework-v1`.
* Confirm that all 'more ***' buttons in feature and simple cards works by clicking the button and confirming the lightbox shows up with correct product name on the header. (Except for CRM which will redirect user to the 
<img width="391" alt="Screen Shot 2022-09-05 at 9 49 00 PM" src="https://user-images.githubusercontent.com/56598660/188465510-01fc023d-b43d-474f-a649-fa6157b951ad.png">

* To confirm if the 'more info' is not visible when the product is not purchasable, go to site specific pricing page that has active jetpack plan.

   eg: [Jurassic Ninja](http://jetpack.cloud.localhost:3000/pricing/jurassic.ninja?flags=jetpack/pricing-page-rework-v1)
 * Confirm that the more info link is not visible for some products that are active to the current plan.
 
<img width="614" alt="Screen Shot 2022-09-06 at 7 09 51 PM" src="https://user-images.githubusercontent.com/56598660/188621638-9873c110-cda0-415e-87f5-359675ae294a.png">





#### Pre-merge Checklist






Related to 1202796695664022-as-1202923891228306

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202923891228306
